File: CSharp/FindTheIndex.cs
using System;

namespace CSharp
{
    // Create a function that finds the index of a given item.
    // https://edabit.com/challenge/L7AwDimWQppT25he6
    public static class FindTheIndex
    {
        public static int Search(int[] arr, int item) => Array.IndexOf(arr, item);
    }
}

File: python/find_the_index.py
"""
Find the Index

Create a function that takes a list and a string as arguments and return the index of the string.

https://edabit.com/challenge/zFmJTah4E9mxJyoFF
"""
from typing import List


def find_index(lst: List[str], txt: str) -> int:
    try:
        return lst.index(txt)
    except ValueError:
        return 0
